goodmap/db/local_json_db.py

```diff
@@ -12,11 +12,15 @@ def __init__(self, config):
         self.data_file_path = config["data_file_path"]
 
     def get_data(self):
-        with open(self.data_file_path, 'r') as file:
-            return json.load(file)
+        try:
+            with open(self.data_file_path, "r") as file:
+                return json.load(file)
+        except FileNotFoundError:
+            with open("example_data.json", "r") as file:
+                return json.load(file)
 
     def save_entry(self, entry):
         data = self.get_data()
-        with open(self.data_file_path, 'w') as file:
+        with open(self.data_file_path, "w") as file:
```

Comment:
What tool did you use to check for formatting? It would be beneficial to use it in pipeline.